superset/app.py

```diff
@@ -549,13 +549,13 @@ def init_app(self) -> None:
         order to fully init the app
         """
         self.pre_init()
+        self.configure_logging()
```

Comment:
let's leave a note to ourselves so we don't make this mistake again

```suggestion
        # Configuration of logging must be done first to apply the formatter properly
        self.configure_logging()
```